src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -977,8 +1075,44 @@ void PolicyHandler::OnActivateApp(uint32_t connection_key,
     }
   } else {
     permissions = policy_manager_->GetAppPermissionsChanges(policy_app_id);
+#ifdef EXTENDED_PROPRIETARY
+    UsageStatistics& usage = app->usage_report();
+
+    usage.RecordAppUserSelection();
+    DeviceParams device_params = GetDeviceParams(
+        app->device(),
+        application_manager_.connection_handler().get_session_observer());
+    permissions.deviceInfo = device_params;
+
+    DeviceConsent consent = policy_manager_->GetUserConsentForDevice(
+        permissions.deviceInfo.device_mac_address);
+    permissions.isSDLAllowed = kDeviceAllowed == consent ? true : false;
+
+    // According to the SDLAQ-CRS-2794, p.9
+    // 'priority' should be ommited in case when device
+    // is not allowed.
+    if (permissions.isSDLAllowed == false) {
```

Comment:
@LuxoftAKutsan just !permissions.isSDLAllowed. At least is used few lines below. In one or other way good to be consistent.